server/utils/mongoCollections.js

```diff
@@ -5,4 +5,6 @@ export const collections = {
   toc: 'toc',
   studyDetails: 'studyDetails',
   users: 'users',
+  dayData: 'assessmentSubjectDayData',
+  dataFile: 'dataFileMetadata',
```

Comment:
We discussed this, so I'm just making a note for posterity. Given that the initial database consolidation will use the same collection names, we can use the `toc` collection for now.